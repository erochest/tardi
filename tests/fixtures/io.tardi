
use: std/fs
use: std/io

// io/write-file
"""
Hello, world!
Goodbye, cruel world!
""" "temporary-file.txt" write-file

// io/read-file
"temporary-file.txt" read-file

// fs/rm
"temporary-file.txt" rm

// io/open

// io/read
// io/read-line
// io/read-lines
// io/read-all

// io/write
// io/write-line
// io/write-lines
// io/write

// fs/exists?
// fs/mkdir
// fs/rmdir
// fs/ensure-dir
// fs/touch

