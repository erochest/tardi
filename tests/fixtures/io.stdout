4213
Hello, world
goodbye cruel world
again
